Makefile

```diff
@@ -39,9 +39,14 @@ SED ?= $(shell which gsed 2>/dev/null || which sed)
 %/$(UPTODATE): %/Dockerfile
 	@echo
 	$(SUDO) docker build --build-arg=revision=$(GIT_REVISION) --build-arg=goproxyValue=$(GOPROXY_VALUE) -t $(IMAGE_PREFIX)$(shell basename $(@D)) $(@D)/
-	$(SUDO) docker tag $(IMAGE_PREFIX)$(shell basename $(@D)) $(IMAGE_PREFIX)$(shell basename $(@D)):$(IMAGE_TAG)
```

Comment:
Why was this removed? I think we need it to keep CI working, am I wrong?

Reply:
It's back now... in single docker build command, with multiple `-t` flags.

Reply:
Reverted it back, due to upcoming change in https://github.com/cortexproject/cortex/pull/4211, which builds images for both platforms, but only tags native one with `$(IMAGE_PREFIX)$(shell basename $(@D))`.

Reply:
I had to revert that change in #4211, as Github Actions don't support `--platform` argument when running docker :(